visidata/loaders/_pandas.py

```diff
@@ -173,13 +215,101 @@ def _selectByILoc(self, mask, selected=True):
         self._checkSelectedIndex()
         self._selectedMask.iloc[mask] = selected
 
+    @asyncthread
+    def selectByRegex(self, regex, columns, unselect=False):
+        '''
+        Find rows matching regex in the provided columns. By default, add
+        matching rows to the selection. If unselect is True, remove from the
+        active selection instead.
+        '''
+        import pandas as pd
+        masks = pd.DataFrame([
+            self.df[col.name].astype(str).str.contains(pat=regex, case=False, regex=True)
```

Comment:
`case=('I' in options.regex_flags)` I think.

Reply:
Good call 👍